### DISCUSSION TITLE
Wrong GraalVM download

### DISCUSSION THREAD

**Discussion by Markospoko** [oracle/graal#8214] [General] [answered]
GraalVM Community Edition for Mac intel link gives ver. 23.1.2
...
...
...

**Comment by alina-yur** [comment]
hi @Markospoko, can you elaborate where you see this? I'm looking at https://github.com/graalvm/graalvm-ce-builds/releases and all builds seem to be GraalVM for JDK 21.0.2 as expected?

**Comment by Markospoko** [comment]
What do you mean by "look" :)? if I go to that releases page all I see with my eyes seems alright, even if I hover over archive name it's alright but when I click to download, I get "graalvm-community-java21-darwin-amd64-23.1.2.tar.gz", the same happens on my second computer. What's weirder, when I click on my Android mobile (I have e-mail app on it and I sometimes use it to quickly peek mail over there), alright file is downloading but I haven't got it fully on the phone, I just wanted to try what it would download.

**Comment by petermz** [comment]
I believe 23.1.2 is the latest version, released about a week ago, so
your version numbers look pretty reasonable to me. What version would
you expect?

**Comment by alina-yur** [comment, marked as answer]
So GraalVM's internal version 23.1.2 corresponds to GraalVM for JDK 21.0.2. There might be some weird caching issue involved, but as long as you @Markospoko have a GraalVM that shows Java version 21.0.2, that's the latest GraalVM CPU release you need.

**Reply by Markospoko** [reply]
I've got an "update": I think I found out that what is downloading is based on network provider I use, if it's my major network provider, the incorrect file is provided but it may not be my ISP doing this for some reason, downloads are alright everywhere else in general, why'd they focus on new JDK for macOS? I tried downloading other files from this release and they seem to be correct 21.0.2 ones, except that file for intel macOS. However I downloaded correct 21.0.2 file for macOS through VPN. Maybe I'm a bit too cautious about that but GraalVM 21? but 23... archive had different size and I decided not to download it, it sort of raised my notice whether it could have malware, are you sure your repository hasn't been hijacked? or maybe Github has been hijacked?

**Reply by ezzarghili** [reply]
Most likely you were facing thise due to some caching issues. you can test the artifacts against the sha256 provided, I am copying them here from my local machine.
`515e3a93acc7e1938daba83eda4272e5495fd302d7cdd99ec7ebf408ed505ab7 graalvm-community-jdk-21.0.2_macos-aarch64_bin.tar.gz`
`7a8aa93fa45d1721908477abf4732a32637d420ffcb66ada9fb6456440b0d9e1 graalvm-community-jdk-21.0.2_macos-x64_bin.tar.gz.sha256`

**Reply by Markospoko** [reply]
I've that the same.
